iphone/Classes/TiUIListItem.m

```diff
@@ -91,6 +94,21 @@ - (void)prepareForReuse
 {
   RELEASE_TO_NIL(_dataItem);
   [super prepareForReuse];
+
+#ifdef USE_TI_UIACTIVITYINDICATOR
```

Comment:
Is this issue not  in TiUITableView ?

Reply:
The issue was only described in Ti.UI.ListView because the developer can re-animate the indicator inside the Ti.UI.TableView as it's children are not hold virtually. As I didn't see any comparable issue on JIRA, I only made it for this one for now (and because we recommend to use Ti.UI.ListView for performant usages).

Reply:
Okey.